bundles/org.openhab.binding.lifx/src/main/java/org/openhab/binding/lifx/internal/LifxLightStateChanger.java

```diff
@@ -257,8 +257,9 @@ private void removeFailedPackets() {
                 while (it.hasNext()) {
                     PendingPacket pendingPacket = it.next();
                     if (pendingPacket.sendCount > MAX_RETRIES && pendingPacket.hasAcknowledgeIntervalElapsed()) {
-                        logger.warn("{} failed (unacknowledged {} times)",
-                                pendingPacket.packet.getClass().getSimpleName(), pendingPacket.sendCount);
+                        logger.warn("{} failed (unacknowledged {} times) to address {}",
+                                pendingPacket.packet.getClass().getSimpleName(), pendingPacket.sendCount,
```

Comment:
Yes the detail is helpful to figure out what light is involved so you can reset it if it is malfunctioning.

I think it would be even more informative if the `logId` is used instead, e.g.:

```java
                        logger.warn("{} failed (unacknowledged {} times) to light {}",
                                pendingPacket.packet.getClass().getSimpleName(), pendingPacket.sendCount, logId);
```

That way it's easy to match this logging with all other trace/debug logging that already uses this `logId`. The `logId` is the MAC or IP address based on how your light is configured. If you've configured your light using MAC only and also use DHCP the IP address in this logging would make it difficult to immediately identify the light if you have many.

Reply:
Ok, i will change that